securedrop-workstation-grsec/debian/postinst

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+# postinst script for essentials
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    configure)
+    # DKMS autoinstall the qubes kernel modules
+    dkms autoinstall 4.14.74-grsec
+    # restart paxctld to set pax flags (incl grub)
+    service paxctld restart
```

Comment:
`service paxctld restart` uses the old SysV-style init syntax. After running this command manually, the paxctld service is indeed started, but it is not marked as "enabled"—meaning it won't start on a subsequent boot. Let's change this line to:

```
systemd enable paxctld
systemd restart paxctld
```

Then it should persist will on subsequent boots.